WeatherToday/Sources/Domain/UseCases/WeatherUseCaseProtocol.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  WeatherUseCaseProtocol.swift
+//
+
+import RxSwift
+import RxCocoa
+import Alamofire
+
+protocol WeatherUseCaseProtocol: AnyObject {
+
+    func getWeather(forCity name: String) -> Single<WeatherApi>
+}
+
+final class WeatherUseCase: WeatherUseCaseProtocol {
+
+    private let weatherRepository: WeatherRepositoryProtocol
+
+    init(weatherRepository: WeatherRepository) {
+        self.weatherRepository = weatherRepository
+    }
+
+    func getWeather(forCity name: String) -> Single<WeatherApi> {
+        return weatherRepository.getWeatherData(cityName: name)
```

Comment:
We can omit return here
```suggestion
       weatherRepository.getWeatherData(cityName: name)
```

Reply:
resolved in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/d19faa8fbaec66b5f882dd4d38f85f4a4ca738c1)